shared/references/cce-redhat-avail.txt

```diff
@@ -430,7 +430,6 @@ CCE-86618-6
 CCE-86619-4
 CCE-86620-2
 CCE-86622-8
```

Comment:
I think we only need a CCE for the main rule, could you remove CEE from two sub-rules?

Reply:
comment incorporated.